packages/user-management-widget/src/lib/mixins/formMixin.ts

```diff
@@ -46,6 +46,7 @@ export const formMixin = createSingletonMixin(
         this.getFormInputs(rootEle).forEach((input) => {
           // eslint-disable-next-line no-param-reassign
           input.value = '';
+          input.removeAttribute('invalid');
```

Comment:
I don't think we should change the components invalid attribute from the driver
Why do we need it?

Reply:
We reset the form, but the validations states remains dirty. This is intended to clear the validation state as well.